tests/FakeItEasy.Tests/Core/CallOrderingContextTests.cs

```diff
@@ -0,0 +1,146 @@
+﻿namespace FakeItEasy.Tests.Core

```

Comment:
Yes. Good point. I'd been (incorrectly) applying my "people shouldn't care about it filter" that I use on  some of the interfaces that are part of the fluent API. Such as `IAfterMustHaveHappenedConfiguration`, which nobody should ever have to talk about. And if they _choose_ to, then that's their business and they can take on the extra `using`.